From bd69685e25bb665a3f9c1d0499f762cb87888aa9 Mon Sep 17 00:00:00 2001
From: Ankit Agrawal <ankit.agarwal@intel.com>
Date: Mon, 12 Aug 2024 15:05:36 +0530
Subject: [PATCH] [WA] Fixed to publish kf4sbl binary.

Publish kf4sbl only for publish target.

Tests: Prepared EB and its booted successfully.

Tracked-On: OAM-123268
Signed-off-by: Ankit Agrawal <ankit.agarwal@intel.com>
---
 tasks/publish.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tasks/publish.mk b/tasks/publish.mk
index 0591355..f582f7f 100644
--- a/tasks/publish.mk
+++ b/tasks/publish.mk
@@ -244,7 +244,7 @@ publish_windows_tools: $(PLATFORM_RMA_TOOLS_CROSS_ZIP)
 	@$(hide) mkdir -p $(publish_tool_destw)
 	@$(hide) $(ACP) $(PLATFORM_RMA_TOOLS_CROSS_ZIP) $(publish_tool_destw)
 else
-publish_ci: publish_liveimage publish_ota_flashfile publish_gptimage_var publish_grubinstaller publish_ifwi publish_kf4sbl publish_firmware_symbols $(PUB_OSAGNOSTIC_TAG) $(PUB_CMCC_ZIP)
+publish_ci: publish_liveimage publish_ota_flashfile publish_gptimage_var publish_grubinstaller publish_ifwi publish_firmware_symbols $(PUB_OSAGNOSTIC_TAG) $(PUB_CMCC_ZIP)
 	$(if $(wildcard $(publish_dest)), \
 	  $(foreach f,$(PUBLISH_CI_FILES), \
 	    $(if $(wildcard $(f)),$(ACP) $(f) $(publish_dest);,)),)
-- 
2.34.1

